web-dev/src/App.js

```diff
@@ -17,43 +17,53 @@ import PrivacyScreen from "./components/PrivacyPage/index.js"
 import {AuthProvider} from "./contexts/auth-context";
 import RootHandler from "./components/RootHandler";
 import UserProfile from "./components/UserProfile";
+import React from "react";
+import { combineReducers, createStore } from "redux";
+import bookmarksReducer from "./reducers/bookmark-reducer.js"
+import { Provider } from "react-redux";
 
 
-// import Netflicks from "./components/index.js";
+const reducer = combineReducers({
```

Comment:
The whole app has access to the bookmarks. This was helpful in displaying the bookmarks in the user profile.